### ISSUE TITLE
[Feature Request]: Publish Release 0.7

### ISSUE THREAD

**Issue by eriqua** [Azure/ResourceModules#1728]
#### To be discussed

Should we 

1. Prefer releasing sooner to fix issues such as outdated API version failing static validation for previous release 0.6 **-OR-** 
2. Postpone the release until all planned issues for milestone 0.7 will be closed, allowing users to work on breaking changes only once?

Main planned breaking changes:
- Folder renaming
   - #1127 
   - #1577 
- Update of GH/ADO secrets
   - #1440 
   - #1605 
   - [POSTPONED TO NEXT RELEASE] #1565 
   - [POSTPONED TO NEXT RELEASE] #1450 
   - [POSTPONED TO NEXT RELEASE] #1085

#### Must have
 
- All module pipelines are green except the ones documented in the [Know Issues Wiki page](https://github.com/Azure/ResourceModules/wiki/Known%20issues). If additional pipelines are agreed to stay red, they need to be added to that page with corresponding reason.

   - [x] #1847
   - [x] #2022

#### Nice to have

- [x] #1254
